src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import core.basesyntax.utils.ParserStringDateToLocalDate;
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    private final ParserStringDateToLocalDate parser = new ParserStringDateToLocalDate();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder();
+        int salaryFull = 0;
+        for (String name : names) {
+            for (int i = 0; i < data.length; i++) {
```

Comment:
```suggestion
        for (String name : names) {
         	int salaryFull = 0;
            for (int i = 0; i < data.length; i++) {
```
так змінну не потрібно буде оновлювати вручну